x-pack/legacy/plugins/alerting/server/lib/alert_instance.test.ts

```diff
@@ -58,29 +58,31 @@ describe('getMeta()', () => {
   });
 });
 
-describe('fire()', () => {
-  test('makes shouldFire() return true', () => {
+describe('enqueue()', () => {
+  test('makes hasSecheduledActions() return true', () => {
     const alertInstance = new AlertInstance({ state: { foo: true }, meta: { bar: true } });
-    alertInstance.replaceState({ otherField: true }).fire('default', { field: true });
-    expect(alertInstance.shouldFire()).toEqual(true);
+    alertInstance.replaceState({ otherField: true }).scheduleActions('default', { field: true });
+    expect(alertInstance.hasSecheduledActions()).toEqual(true);
   });
 
-  test('makes getFireOptions() return given options', () => {
+  test('makes getSechduledActionOptions() return given options', () => {
     const alertInstance = new AlertInstance({ state: { foo: true }, meta: { bar: true } });
-    alertInstance.replaceState({ otherField: true }).fire('default', { field: true });
-    expect(alertInstance.getFireOptions()).toEqual({
+    alertInstance.replaceState({ otherField: true }).scheduleActions('default', { field: true });
+    expect(alertInstance.getSechduledActionOptions()).toEqual({
       actionGroup: 'default',
       context: { field: true },
       state: { otherField: true },
     });
   });
 
-  test('cannot fire twice', () => {
+  test('cannot enqueue for firing twice', () => {
```

Comment:
`cannot schedule actions for execution twice`, if we want to remove enqueue terminology?